### PR TITLE
Removed unwanted escape chars in reports' bundles

### PR DESCRIPTION
**Pull request by mwithi**
Also, fixed some translations and UTF-8 encodings

### CODE REVIEW

File: rpt/PharmaceuticalStockWard.properties [hunk 1]
```diff
@@ -4,15 +4,11 @@ AvgValue = AVG VALUE
...
 Date = Date
 
-FootNotes1 = Value estimation using the unit costs average transferred to this \
-             ward for the given product
+FootNotes1 = Value estimation using the unit costs average transferred to this ward for the given product
 
-FootNotes2 = Value estimation using the unit costs weighted average \
-             transferred to this ward for the given product (bigger quantities \
-             contributes more)
+FootNotes2 = Value estimation using the unit costs weighted average transferred to this ward for the given product (bigger quantities contributes more)
```
Comment (dbmalkovsky) [inline rpt/PharmaceuticalStockWard.properties:9]: `(bigger quantities contributes more)` should be `(bigger quantities contribute more)`.   That is `contribute` should be singular.

File: rpt/PharmaceuticalStockWard_en.properties [hunk 2]
```diff
@@ -4,15 +4,11 @@ AvgValue = AVG VALUE
...
 Date = Date
 
-FootNotes1 = Value estimation using the unit costs average transferred to this \
-             ward for the given product
+FootNotes1 = Value estimation using the unit costs average transferred to this ward for the given product
 
-FootNotes2 = Value estimation using the unit costs weighted average \
-             transferred to this ward for the given product (bigger quantities \
-             contributes more)
+FootNotes2 = Value estimation using the unit costs weighted average transferred to this ward for the given product (bigger quantities contributes more)
```
Comment (dbmalkovsky) [inline rpt/PharmaceuticalStockWard_en.properties:9]: again; `contributes` -> `contribute`

File: rpt/hmis108_referrals.properties [hunk 3]
```diff
@@ -6,16 +6,11 @@ Count = Count
...
 
 Month = Month:
 
-NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM \
-                                             this unit " are inpatient \
-                                             discharged the selected month.\n- \
-                                             the "referred TO this unit " are \
-                                             inpatient admitted the selected \
-                                             month.
+NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM this unit " are inpatient  discharged the selected month.\n- the "referred TO this unit " are inpatient admitted the selected month.
```
Comment (dbmalkovsky) [inline rpt/hmis108_referrals.properties:13]: should there be a space after `unit` and before the quote?  Happens twice.

File: rpt/hmis108_referrals_en.properties [hunk 4]
```diff
@@ -6,16 +6,11 @@ Count = Count
...
 
 Month = Month:
 
-NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM \
-                                             this unit " are inpatient \
-                                             discharged the selected month.\n- \
-                                             the "referred TO this unit " are \
-                                             inpatient admitted the selected \
-                                             month.
+NoteReferredFromThisUnitReferredToThisUnit = *Note: \n- the "referred FROM this unit " are inpatient  discharged the selected month.\n- the "referred TO this unit " are inpatient admitted the selected month.
```
Comment (dbmalkovsky) [inline rpt/hmis108_referrals_en.properties:13]: same question as immediately above for line 13

**Comment by dbmalkovsky** [comment]
Few minor comments; other that that LGTM

File: rpt/PharmaceuticalStockWard.properties (hunk 1 above)
Comment (mwithi) [reply rpt/PharmaceuticalStockWard.properties:9]: Thanks!

File: rpt/hmis108_referrals.properties (hunk 3 above)
Comment (mwithi) [reply rpt/hmis108_referrals.properties:13]: yep, thanks!

**Pull request merged**